templates/download/xilinx/index.html

```diff
@@ -37,16 +37,16 @@ <h1>Install Ubuntu on Xilinx</h1>
     <div class="p-tabs">
       <div class="p-tabs__list js-tabbed-content" role="tablist" aria-label="Install xilinx">
         <div class="p-tabs__item">
-          <button class="p-tabs__link" role="tab" aria-selected="true" aria-controls="zynq-ultrascale-tab" id="zynq-ultrascale">Xilinx Zynq UltraScale+ MPSoC
-            Development Boards</button>
+          <button class="p-tabs__link" role="tab" aria-selected="true" aria-controls="kira-kv260-vision-ai-tab" id="kira-kv260-vision-ai" tabindex="-1">Xilinx
```

Comment:
`tabindex="-1"` should remain on the second tab, at the moment if you tab to this section, it automatically focuses on the second tab, and you have to press the left arrow key to get to the first.